src/utils/storage.js

```diff
@@ -0,0 +1,22 @@
+const os = require('os');
+const path = require('path');
+const { Store } = require('data-store');
+
+const getStoredValues = (templateName) => {
+  const store = new Store({
+    path: path.join(os.homedir(), `.create-using-template/${templateName}.json`),
+  });
+  return store.get();
+};
+
+const storeConfiguration = (templateName, values) => {
```

Comment:
```suggestion
const setStoreValues = (templateName, values) => {
```

Reply:
resolved in 26a819b549befc016bc6480f3e074b9597bc3a12